app/src/main/java/com/infomaniak/mail/ui/main/folder/TwoPaneFragment.kt

```diff
@@ -175,6 +176,20 @@ open class TwoPaneFragment : Fragment() {
         }
     }
 
+    override fun onConfigurationChanged(newConfig: Configuration) {
+        super.onConfigurationChanged(newConfig)
+        ensureThreadIsOpen()
+    }
+
+    override fun onResume() {
+        super.onResume()
+        ensureThreadIsOpen()
+    }
+
+    private fun ensureThreadIsOpen() {
+        if (mainViewModel.isInThread) openThread(mainViewModel.currentThreadUid.value!!)
```

Comment:
```suggestion
        mainViewModel.currentThreadUid.value?.let { currentThreadUid ->
           if (mainViewModel.isInThread) openThread(currentThreadUid)
        }
```

Reply:
The whole purpose of `isInThread ` is to avoid writing that :D

Reply:
Yes but `!!` are evil. Don't be so close to the dark side of the force.